test/performance/odp_random.c

```diff
@@ -14,8 +14,32 @@
 #include <odp_api.h>
 #include <odp/helper/odph_api.h>
 
+#define PSEUDO_RANDOM (-1)
+
 #define MB (1024ull * 1024ull)
 
+typedef struct test_global_t test_global_t;
+
+typedef struct thread_arg_t {
+	test_global_t *global;
+	int thread_idx;
+	uint8_t *data;
+
```

Comment:
Extra newline.

Reply:
I prefer this style where the name of the type/struct is easier to find after the list of struct members.

Reply:
OK, maybe same treatment to `options_t`.

Reply:
Done in v2.